package.json

```diff
@@ -60,6 +60,7 @@
     "moment": "^2.22.2",
     "node-sass": "^4.10.0",
     "nodemon": "^1.18.7",
+    "pouchdb": "^7.0.0",
```

Comment:
You shouldn't need `pouchdb` here at all, you can just use core, http and (if you need it, and I'm not sure you do) mapreduce manually:

```
const PouchDB = require('pouchdb-core');
PouchDB.plugin(require('pouchdb-adapter-http'));
PouchDB.plugin(require('pouchdb-mapreduce'));
```

(we want to avoid requiring `pouchdb` as much as humanly possible, because it comes with lots of dependencies, including annoying ones that require native compilation and things like that)